package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "react-spa",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1",
+    "react-hot-loader": "^3.0.0-beta.7",
+    "react-scripts": "1.0.11"
+  },
+  "scripts": {
+    "dev": "react-scripts build --NODE_ENV=development&& webpack --config webpack.config.js && webpack-dev-server",
```

Comment:
Try to get rid of `react-scripts` command, as if you not using create-react-app at all. So that `npm run dev` sets `NODE_ENV`, starts webpack and dev server.